spyder/plugins/completion/providers/languageserver/provider.py

```diff
@@ -837,7 +837,7 @@ def generate_python_config(self):
         # Setup options in json
         python_config['cmd'] = cmd
         if host in self.LOCALHOST and not stdio:
-            python_config['args'] = ('--host {host} --port {port} --tcp '
+            python_config['args'] = (f'--host {host} --port {port} --tcp '
```

Comment:
This arguments are meant to be replaced by a `format` call here:

https://github.com/spyder-ide/spyder/blob/431a42152d9b8bce4f98d1eb191f2f5d8a986d91/spyder/plugins/completion/providers/languageserver/client.py#L206-L208